pepsico/isimiptozarr.py

```diff
@@ -29,6 +29,14 @@ def set_up_dims(xda, time_res="daily", time_dim=None, lon_dim="Lon", lat_dim="La
     time_res : str, optional
         indicates the time resolution of the set of time-dependent files.
         Default is "daily" and other option is "dekadal"
+    time_dim : str, optional
+        indicates name of time dimension in xda, to be renamed "T"
+    lon_dim : str, optional
+        indicates name of longitude dimension in xda, to be renamed "X"
+        Default is "Lon"
+    lat_dim : str, optional
+        indicates name of latitude dimension in xda, to be renamed "Y"
+        Default is "Lat"
```

Comment:
If the input data has CF standard_names, we should probably use those to identify the lat, lon, and time dimensions. If it doesn't, then these arguments should become mandatory, and we should add appropriate standard_names to the output.

Just thinking out loud, doesn't have to be done now if you're in a hurry. In any case, there's no point doing this kind of generalization under pepsico, only when it moves to the top level.